TUMFARSynchrony/experimental-hub
# Asymmetric Filters Feature

# Overview
This PR contains the asymmetric filters feature. This feature makes it possible to activate the asymmetric filters for each participant and change the filters through which that participant sees other participants.

## How to test?
Create an experiment with at least 2 participants. Navigate to the "Asymmetric Filters" tab within the participant data modal for a participant and enable the asymmetric filters. From the appeared list, apply filters to the other participants for this participant's point of view. Save the session and start the experiment to see the asymmetric filters applied for that participant.

# Checklist
